### ISSUE TITLE
Lua output as a Logical Switch input is not permanent in EdgeTX 2.9.1

### ISSUE THREAD

**Issue by boomerwing** [EdgeTX/edgetx#4230]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

I am using an Lua script which I call "sayalt". This script just reads the current altitude value, checks to see if it is close to any 100 ft. step in altitude, and outputs a flag to activate a logical switch. The logical switch uses another Edge Logical Switch and Special Function to make a voice announcement of the current altitude which has just been flagged.
The Lua script works fine when the first Logical switch has the Lua script output "flg" as input. But the problem is, Every time I start the new model I have to go into the Logical switches and add the "flg" as input to the Logical Switch.
In Open TX on a Taranis, once I had defined the Logical Switch it stayed defined.

I am using a Radiomaster TS16S with EdgeTX 2.9.1.

Lua Script sayalt.lua
```lua
local outputs = {"flg"}
     --[[ This is seen as LUA1a in the Companion Logical Switch setup screen when you select input
          for the Logical Switch if your LUA script is LUA1
	   ]]

 local function run_func()
		  altNow = getValue("Alt")
		  modVal = altNow%100   --  look for 100 ft increments.  You can change this value
		  if modVal < 3 or modVal > 97 then  --  between x97 and y03
		     flg = 1024
		  else
		     flg = -1024
		  end
	   return flg
end
return {run=run_func, output=outputs}
```

### Expected Behavior

I expect the definition of the Logical Switch to be permanent.  I do not want to fix the definition every time I load the model.

### Steps To Reproduce

Each time I load a new model I have to redefine the Logical Switch.

### Version

2.9.1

### Transmitter

Radiomaster TX16S / TX16SMK2

### Operating System (OS)

Windows

**Comment by boomerwing** [comment]
I have installed EdgeTX 2.9.2 which was supposed to have fixed this problem.  When I turn the RadioMaster TX16S OFF then ON again  the  problem is still there.  The output value of the Lua script does not remain as the input of the Logical Switch.  Every time I turn on the TX I must go into the Logical Switch and correct the entry before I fly my model.

**Comment by philmoz** [comment]
The fix was added to 2.10; but it doesn't look like it was added to 2.9.

**Comment by boomerwing** [comment]
OK.  I thought I saw in the description of 2.9.2 that it was included.
I do have a work-around until 2.10 if that is the decision.  I will watch for the fix.
